tests/xtro-sharpie/iOS-Speech.todo

```diff
@@ -1,2 +0,0 @@
-!deprecated-attribute-missing! SFSpeechRecognitionRequest::interactionIdentifier missing a [Deprecated] attribute
-!deprecated-attribute-missing! SFSpeechRecognitionRequest::setInteractionIdentifier: missing a [Deprecated] attribute
```

Comment:
I think you should be able to delete this file if there is not anything else in there!

Reply:
I had no idea - thanks for catching this!